### ISSUE TITLE
Uninitialized Constant Api::V2::MyController

### ISSUE THREAD

**Issue by mark100net** [jwoertink/jko_api#9]
I have just gotten around to needing a V2 and have run into a problem. I am either misunderstanding something or am missing something.

Based on the docs when I create a V2 I do not have to create a V2 controller file for every controller (which as I understood it was the purpose of the gem). Just the ones that need to change. Is that correct?

I created version 2 by adding just `version 2` to the routes file. Now if a client is on version 2 it is generating an Uninitialized Constant on any controller that does not have a V2 file.

Do we need to create an empty V2 descendent for every controller in every new version?

**Comment by jwoertink** [comment]
You are correct in that is the purpose of the gem. You shouldn't need to create anything extra other than telling the routes that there is a v2. However, until just a week ago, I haven't touched rails 5. I've sort of made updates based on other people using rails 5. Now that I'm using it, I'm seeing some updates that need to be done. I'll try and figure out some fixes, but if you already see some issues, I'll gladly accept some PRs.

**Comment by mark100net** [comment]
I wasn't able to get it working other than by making an empty controller file that inherits from the previous version for each controller. I had to settle for that so that I could continue but if I get a chance I'll see if I can figure out a better fix.

**Comment by jwoertink** [comment]
ah, ok. My guess is that it's something with [this](https://github.com/jwoertink/jko_api/blob/master/lib/jko_api/class_descendants_builder.rb#L50). It existed in rails 4.2, but was ripped out in 5. I just copied it straight over. I'll mess around with it too and see if I can figure out the issue.
